Community.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: default
+title: Community
+permalink: /community/
+nav_order: 99
+---
+
+Slack Open Source Firmware
+===
+
+* [Register to Open Source Firmware](https://slack.osfw.dev/)
```

Comment:
"Register to Slack's Open Source Firmware" ?

Reply:
Sure, I just copied the existing links

https://github.com/osresearch/heads-wiki/commit/29d8940f12215f3fba1f3b5f414a406e127efd22#diff-d680e8a854a7cbad6d490c445cba2eba